### ISSUE TITLE
The LOGIN Page is not Redirecting to Registration Page

### ISSUE THREAD

**Issue by gurupreetkaushik7** [bhglove/SimpleNewsReader#1]
Hello,

I've just ran your code on Android Studio. 

I've been through your code quite thoroughly. When a NEW user enters an email ID and password, it does not redirect to the registration page. The result is obviously -1 in Async Task. 

I need a way to go to the Registration activity once the result is -1 for new users. 

Please help.
Thanks

**Comment by bhglove** [comment]
Hi! First, thank you for looking at my code. I'm not sure how much of a resource I can be, because I'm still a novice android programmer. But the reason why the registration activity isn't popping up because the script is using the http protocol and not the https protocol. The server that I'm using removed the former protocol and sends an error response instead of the results from my php script. 

In other words, replace the database url with your own php script to handle authentication. 

I hope this helps.`[ Here is a more experienced resource. ](http://www.androidhive.info/2012/01/android-login-and-registration-with-php-mysql-and-sqlite/)`
